autoload/go/fix.vim

```diff
@@ -0,0 +1 @@
+/Users/fatih/go/src/github.com/fatih/goautofix/vim/fix.vim
```

Comment:
This looks like it was mistakenly committed.

Reply:
Yeah, it was symlinked (I was lazy) from a personal project for a small tool I wrote and wanted to try out. Removed them :)